usb/device/targets/TARGET_NORDIC/TARGET_MCU_NRF52840/USBPhyHw.h

```diff
@@ -0,0 +1,111 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018-2019 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef USBPHYHW_H
+#define USBPHYHW_H
+
+#include "mbed.h"
```

Comment:
include what is required, not mbed.h

Reply:
Ah, broke my own personal rule. Copy-pasted the template Phy. 

Also, I'm rerunning all USB tests at the moment and I'll post back the results.